github/ci/prow-deploy/kustom/overlays/workloads-production/resources/priority-classes.yaml

```diff
@@ -3,8 +3,8 @@ apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
   name: sriov
-value: 1000000
-preemptionPolicy: Never
+value: 1000001
```

Comment:
Will this be properly updated on the cluster?

Reply:
Hpefully #2087 should handle this situation